crypto/pem/pem_local.h

```diff
@@ -95,14 +95,22 @@
 # define IMPLEMENT_PEM_provided_write_body_fallback_cb(str, asn1,       \
                                                        writename)       \
     legacy:                                                             \
-    return PEM_ASN1_##writename((i2d_of_void *)i2d_##asn1, str, out,    \
-                                x, enc, kstr, klen, cb, u)
+    return PEM_ASN1_##writename##((i2d_of_void *)i2d_##asn1, str, out,  \
+                                  x, enc, kstr, klen, cb, u)
 
 # define IMPLEMENT_PEM_provided_write_to(name, type, str, asn1,         \
```

Comment:
There is no propq in the macro params?

Reply:
Why should there be?  `IMPLEMENT_PEM_provided_write_body_vars()` implements the two variables `libctx` and `propq`, which the rest of the building blocks can use.  Why should the implementing macros need to specify those variables explicitly?

Reply:
Just a personal style thing I guess.

Reply:
That seems unnecessary, and potentially confusing...

Reply:
That is your opinion.